src/commands/commit-and-tag-version.ts

```diff
@@ -0,0 +1,69 @@
+import chalk from 'chalk';
+
+import { getChangelogText } from './internal/create-changelog';
+import { getGitBranch, gitAdd, gitCommit, gitPush, gitPushTags, gitTag } from '../git/git';
+import { getPackageVersion, getPackageVersionTag } from '../versions/package_version';
+import { getPrevVersionTag } from '../versions/git_helpers';
+import { sh } from '../git/shell';
+
+const BADGE = '[commit-and-tag-version]\t';
+
+/**
+ * Commits, tags and pushes the current version
+ * (when on one of the applicable branches).
+ *
+ */
+export async function run(...args): Promise<boolean> {
+  const isDryRun = args.indexOf('--dry') !== -1;
+  const isForced = process.env.CI_TOOLS_FORCE_PUBLISH === 'true' || args.indexOf('--force') !== -1;
+
+  printInfo(isDryRun, isForced);
+
+  const packageVersion = getPackageVersion();
+  const changelogText = await getChangelogText(getPrevVersionTag());
+  const commitSuccessful = commitPushAndTagCurrentVersion(packageVersion, changelogText);
```

Comment:
Little nitpicking here, but this would fit more, imho:

```suggestion
  const commitSuccessful = pushCommitAndTagCurrentVersion(packageVersion, changelogText);
```

After all, one does not commit a push, but one always pushes a commit. 😉